docs/data-sources/hcd.md

```diff
@@ -0,0 +1,32 @@
+The High Cost Drugs (HCD) data is a dataset containing specialist medicines prescribed by hospitals to patients for the management of long term conditions. 
+
+## Metadata
+
+* **Data provider** North East Commissioning Support Unit (NECS) with approvals from NHS Digital.
+* **Participation / Coverage** Hospital prescribing of [high cost drugs](https://www.england.nhs.uk/publication/nhs-england-drugs-list/) across all 135 CCGs nationally.
+* **Update frequency in OpenSAFELY** Data covering April 2018 to March 2020 is currently available in OpenSAFELY, with no updates planned. 
```

Comment:
suggest adding at end of this bullet (see discussion section of paper for our proposal)

Reply:
will add in now